.gitignore

```diff
@@ -1,3 +1,5 @@
 /vendor/
 composer.lock
-.php_cs.cache
\ No newline at end of file
+.php_cs.cache
+.DS_Store
+/layers/newrelic/.DS_Store
```

Comment:
These should not be here. 

You should add `.DS_Store` in your global gitignore.